README.md

```diff
@@ -264,6 +264,14 @@ specified first. Also, `--check` ignores the value of the `controller` and assum
 will be present during a real run. That means, in presence of an invalid `controller`,
 `--check` may signal success but the actual play run fails.
 
+When destroying and removing the bonding profiles, the order of the `connections` list
+also matters. If only specifying the `controller` profile in the `connections` list and
+bringing it down, the port profiles will be brought down automatically. However, if not
+all the port profiles are brought down, the connection state of the `controller`
+profile will not be affected (stay `up`). Therefore, when it comes to destroy and
+remove the bonding profiles utterly, it is recommended to bring down and remove all the
+port profiles first, then bring down and remove the controller profile.
```

Comment:
Why do the port profiles need to be removed first? From the text before that it seems it would be enough to destroy all profiles at the same time.

Reply:
Right, it is true that bringing down the controller profile only will necessarily and automatically bringing down the port profile. In terms of destroying the bonding connection,  the port network connection dictionary and the controller network connection dictionary can be defined in the same network connections list, the "down" action will be executed before "absent" action, as a result,  all profiles in the bonding connection can be and destroyed and removed at the same time.